### PR TITLE
:sparkles: Homepage

### PR DESCRIPTION
**Pull request by mauromorales**

### CODE REVIEW

File: static/js/trees.js [hunk 1]
```diff
@@ -0,0 +1,581 @@
+const DOWNLOAD_CARDS_TREE = [
+    {
+        integration:"ubuntu",
+        links: {
+            "X86": "https://github.com/kairos-io/provider-kairos/releases/download/v2.3.1/kairos-ubuntu-v2.3.1-k3sv1.25.11+k3s1.iso",
```
Comment (mauromorales) [inline static/js/trees.js:5]: This being static might be a problem, because we need the url to be generated

File: layouts/index.html [hunk 2]
```diff
@@ -13,62 +13,84 @@
...
             </div>
+
+            <div class="hero-logo astro-7XAARZHW">
+              <img src="/index/armadillo.png" alt="main logo" width="318" class="astro-7XAARZHW">
+              <div class="hero-latest-release">
+                Check out
+                <p>our latest release</p>
+                <div class="hero-latest-release-version">
+                  <div>Latest release</div>
+                  <div>{{$.Page.Site.Params.softwareVersions.kairos_version}}</div>
```
Comment (mauromorales) [inline layouts/index.html:80]: Update variable to latest

File: assets/scss/_styles_project.scss [hunk 3]
```diff
@@ -374,3 +600,681 @@ footer:where(.astro-7XAARZHW) {
...
+    }
+    .press-cards > div > p{
+        height: 100%;
+    }
+    .makes-us-diff > h2{
+        font-size: 36px;
+    }
+    .makes-us-diff-cards > div img{
+        margin-left: -25px;
+        margin-bottom: -30px;
```
Comment (mirceabozi) [inline assets/scss/_styles_project.scss:1242]: Note to self: Delete this line.

Comment (mauromorales) [reply assets/scss/_styles_project.scss]: done in 9b7d6f1

File: static/js/trees.js [hunk 4]
```diff
@@ -0,0 +1,581 @@
...
+    },
+    {
+        content: "Check us out in this The New Stack article by our maintainer, Ettore di Giacinto",
+        imgSrc: "/index/press-blade/theNewStack.png",
+        link: "https://thenewstack.io/livin-kubernetes-on-the-immutable-edge-with-kairos-project/",
+    },
+    {
+        content: "Learn how to use the famous OSS project, LocalAI, with Kairos and K3s on your nodes",
+        imgSrc: "/index/press-blade/k8s-superpowers.png",
+        link: "https://kairos.io/docs/examples/localai/",
```
Comment (mauromorales) [inline static/js/trees.js:239]:
```suggestion
        link: "https://thenewstack.io/looking-for-a-k3os-alternative-choosing-a-container-os-for-edge-k8s/",
```

**Comment by mauromorales** [comment]
Adding these here just so they don't get lost between the two PRs

- https://github.com/kairos-io/kairos-docs/pull/70#issuecomment-1673478784
- https://github.com/kairos-io/kairos-docs/pull/70#issuecomment-1673483710

cc @mirceabozi

File: layouts/index.html (hunk 2 above)
Comment (mauromorales) [reply layouts/index.html:80]: done

**Pull request merged**